src/socketio/transport.rs

```diff
@@ -383,7 +385,12 @@ mod test {
         };
 
         socket
-            .emit_with_ack("test".into(), "123", Duration::from_secs(10), ack_callback)
+            .emit_with_ack(
+                "test".into(),
+                "123",
+                Duration::from_secs(10),
+                ack_callback,
+            )
```

Comment:
Could you run `cargo fmt` another time? This gets linted, but it would be nice to keep the checks green ;)